backend/src/database.ts

```diff
@@ -109,28 +109,28 @@ export type dbGameSummary = {
   numMoves: number;
 };
 
-export type dbFinishedGameWithoutEloId = Omit<dbFinishedGame, "eloIds">;
+export type dbFinishedGameWithoutIdTokens = Omit<dbFinishedGame, "idTokens">;
 
 // ============================================
 // Functions to interact with the database.
 // ============================================
 
-export async function getPseudoPlayer(
-  eloId: string
-): Promise<dbPseudoPlayer | null> {
+function isGuest(idToken: string): boolean {
+  return idToken.substring(0, 6) !== "Auth0|";
```

Comment:
If the string "Auth0|" appears or could appear anywhere else, we should probably capture it as a global constant.